src/python/e2e-test-runner/e2e_test_runner/test_consul_dns.py

```diff
@@ -3,7 +3,11 @@
 """
 import socket
 
+import pytest
 
+
+# TODO re-enable this after the fix to Nomad DNS is released
```

Comment:
A link to an issue, or indication of expected Nomad version would be useful here.

Reply:
Added. The fix is in main, just waiting for the next version release of Nomad